frontend/src/views/App/App.tsx

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
+import { AppRoutes } from 'routes/AppRoutes';
 import { StyledComponent } from './App.styled';
 
 export const App = () => {
   return (
     <div className="App">
       <StyledComponent>
```

Comment:
To StyledComponent możemy chyba wyrzucić - dodałem jako przykład dla Was

Reply:
Możę byłabyś w stanie wykorzystać tutaj layout ? https://ant.design/components/layout/

Np taki?
![image](https://user-images.githubusercontent.com/41823689/166156967-d4cc3951-be22-45cf-a119-07efa59d1893.png)

Reply:
I wtedy HeaderPage który stworzyłaś chcielibyśmy wykorzystać w "Header".
o i router dodamy już w "content".

Dasz radę coś z tego ukuchcić?